### PR TITLE
Swapping the order of kind and dataset_id in Query constructor.

### PR DESCRIPTION
**Pull request by dhermes**
Also updating docstring in Query.fetch() to reflect this.

Partially addresses #704.

### CODE REVIEW

File: gcloud/datastore/query.py [hunk 1]
```diff
@@ -29,13 +29,13 @@ class Query(object):
...
-    :type kind: string
-    :param kind: The kind to query.
-
     :type dataset_id: string
     :param dataset_id: The ID of the dataset to query.  If not passed,
                        uses the implicit default.
 
+    :type kind: string
+    :param kind: The kind to query.
+
```
Comment (tseaver) [inline gcloud/datastore/query.py]: Why doesn't the order in the docstring match the order in the declaration?

**Comment by tseaver** [comment]
LGTM, modulo the `datastore.` prefix notation issues I expressed earlier.

File: gcloud/datastore/query.py (hunk 1 above)
Comment (dhermes) [inline gcloud/datastore/query.py]: D'oh! I blindly changed it because I thought it needed to move. Turns out it was wrong before.

**Comment by dhermes** [comment]
Rebased on top of HEAD / swapped the docstring args back to the rightful position.

Will wait on Travis before merging. Boo to slow Travis. It [seems][1] they went from no outages to weekly outages in the recent past.

[1]: https://twitter.com/traviscistatus

**Comment by dhermes** [comment]
Merging. Coveralls decided not to send the message but it says 100%:
https://coveralls.io/builds/2102263

**Pull request merged**